### DISCUSSION TITLE
pyproject.toml: package_data?

### DISCUSSION THREAD

**Discussion by nschloe** [pypa/setuptools#3209] [Q&A] [answered]
In `setup.cfg`, one could add package data via
```
[options.package_data]
* = *.txt, *.rst
hello = *.msg
```
(See [here](https://setuptools.pypa.io/en/latest/userguide/declarative_config.html).)

How to do the same in the new pyproject.toml-based builds? (`package_data` is not part of PEP 621.)

**Comment by abravalheri** [comment, marked as answer]
Hi @nschloe the equivalent should be `[tool.setuptools.package-data]`.

There is some documentation in https://validate-pyproject.readthedocs.io/en/latest/schemas.html about what are the valid fields.

Note that I am currently preparing a fix for `include-package-data` so I don't know if `package-data` was also affected.

**Reply by nschloe** [reply]
Yup, that works. Thanks for the link!
